pytorch_lightning/trainer/connectors/signal_connector.py

```diff
@@ -36,7 +36,7 @@ def register_signal_handlers(self) -> None:
         sigterm_handlers: List[Callable] = []
 
         if _fault_tolerant_training():
-            sigusr1_handlers.append(self.fault_tolerant_sigusr1_handler_fn)
+            sigterm_handlers.append(self.fault_tolerant_sigterm_handler_fn)
```

Comment:
you missed updating the actually SIGUSR1, which I believe was the indention of this PR.
remember to also update all references to "SIGUSR1" in this file, and there are few others in other files in Lightning.

Reply:
Hey @awaelchli,

I am not sure to fully understand your comment. `sigusr1` is still use for Slurm and I believe all reference left are SLURM related.